packages/core/src/internal-components/panel/panel.element.ts

```diff
@@ -32,13 +31,15 @@ export const CdsInternalPanelTagName = 'cds-internal-panel';
  * @cssprop --box-shadow
  */
 export class CdsInternalPanel extends LitElement {
+  @id() private panelId: string;
+
   static get styles() {
     return [baseStyles, styles];
   }
 
   render() {
     return html`
-      <div class="private-host">
+      <div class="private-host" role="region" aria-labelledby="${this.panelId}">
```

Comment:
@mathisscott We chat about the `role` yesterday but I forgot to ask you about the `aria-labelledby`. I look around in the `core` package and I saw we used `@id()` for aria labels and aria descriptions. Let me know if I missed something here and I have to go in another direction.

Reply:
The `aria-labelledby` will need to point to a header or other html block element provided by the end user. Likewise, the id will be provided on their end as well. You probably want a property that then projects its value (a string id) down into the private host wrapper.

Reply:
The aria labelledby and role need to exist on the host element. A aria-labelledby in the shadow dom cannot reference an id in the light DOM. This is because `id` values are unique to each. The role can be applied to the host similar to the alert component

```typescript
@property({ type: String }) role = 'region';
```

Then apps can assign the aria labelledby on the host

```
<cds-card aria-labelledby="title"> <!-- role will be auto set to this element -->
  <h3 id="title"></h3>
</cds-card>
```